apps/omg_watcher_rpc/test/omg_watcher_rpc/web/controllers/account_test.exs

```diff
@@ -128,7 +128,13 @@ defmodule OMG.WatcherRPC.Web.Controller.AccountTest do
          }}
       ])
 
-      assert WatcherHelper.get_exitable_utxos(alice.addr) == WatcherHelper.get_utxos(alice.addr)
+      # Drop watcher_info's creating_txhash and spending_txhash before comparing to security's exitable utxos
```

Comment:
Also same as before - fix it for good

Reply:
Addressed in https://github.com/omisego/elixir-omg/pull/1224/commits/0943dbe1fd8a9874f716e0be44f8c9c958ed9d0d